coral-hive/src/test/java/com/linkedin/coral/hive/hive2rel/CoralTestUDF.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2018-2020 LinkedIn Corporation. All rights reserved.
+ * Copyright 2018-2021 LinkedIn Corporation. All rights reserved.
```

Comment:
The spotlessApply task only checks/updates files that have changed since origin/master. Files like this where only the copyright date was modified should be backed out.

Reply:
Reverted.